### PR TITLE
Updated how translation gets loaded, made a constant for counts color

### PR DESCRIPTION
**Pull request by SanjeevLakhwani**
- Now translation is loaded with i18n-http-backend
- Each useTranslation hook now specified its namespace scope
- new color for counts is now a constant instead of being repeated at multiple places

### CODE REVIEW

File: src/js/constants/configConstants.js [hunk 1]
```diff
@@ -5,3 +5,10 @@ export const publicOverviewUrl = '/overview';
 export const searchFieldsUrl = '/fields';
 export const katsuUrl = '/katsu';
 export const provenanceUrl = '/provenance';
+
+export const translationEnUrl = '/public/locales/en/translation_en.json';
+export const translationFrUrl = '/public/locales/fr/translation_fr.json';
```
Comment (gsfk) [inline src/js/constants/configConstants.js:10]: These both look like dead code to me. Isn't this handled by the 118n `backend.loadpath`?

**Review by gsfk — COMMENTED** [review]
Unable to test non-default translations locally. Any suggestions? Otherwise a few line-by-line comments.

File: src/public/locales/fr/default_translation_fr.json [hunk 2]
```diff
@@ -43,5 +43,6 @@
   "Manage Charts": "Gestion des tableaux",
   "Insufficient data available.": "Pas suffisamment de données.",
   "Terms of Use": "Conditions d'utilisation (en anglais)",
-  "No Data": "Pas de données"
+  "No Data": "Pas de données",
+  "Counts": "Comptes"
```
[the remark this answers is not in the document]
Comment (dbujold) [reply src/public/locales/fr/default_translation_fr.json:47]: Termium suggests "Données d'énumération", it's a bit funny but perhaps we should stick to that.

[the remark this answers is not in the document]
Comment (SanjeevLakhwani) [reply src/public/locales/fr/default_translation_fr.json:47]: I would confirm with David__

File: src/js/constants/configConstants.js (hunk 1 above)
Comment (SanjeevLakhwani) [reply src/js/constants/configConstants.js:10]: ooops yes

File: src/public/locales/fr/default_translation_fr.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (davidlougheed) [reply src/public/locales/fr/default_translation_fr.json:47]: who, the david above you?

**Pull request merged**